packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.jsx

```diff
@@ -424,6 +424,10 @@ const ProofOfAddressForm = ({
                                             setDocumentFile({ files: df.files, error_message: df.error_message })
                                         }
                                         getSocket={WS.getSocket}
+                                        settings={{
+                                            document_issuing_country:
+                                                account_settings?.citizen ?? account_settings?.country_code,
```

Comment:
can we somehow improve this part of code?
?? operator is applied also for empty string, but I guess we don’t expect that account_settings?.citizen can be empty  string. 
and what if citizen is truthy value, but there is no country_code. Should we assign something to document_issuing_country?

Reply:
We're using the same condition in CFD and we're following the same condition we've and it has never failed for in any case. 
Citizen can never be empty as a user must be a citizen of a country in order to create an account if I'm not wrong. 

Hope this answers the question

Reply:
Alright , but anyway, this operator '??' means that we can assign empty string.  so if are not supposed to pass any empty string as well as other falsy values, I believe we should use '||'.
but it’s up to you :)
thanks for the answer

Reply:
@shahzaib-deriv do we need to pass country code for proof of address? is this expected?

Reply:
@amina-deriv Yes, in a way it's a failsafe as we need to track a country if the citizen isn't available a user may select documents from the country he's residing in as a proof.

Reply:
Also citizen can be empty for old clients. Recently only we made citizenship mandatory during onboarding. There can be clients whose citizenship is not yet set.

Reply:
But proof of address and proof of ownership doesn't have country selection option. right?

Reply:
In my understanding, the both `citizen` & `country_code` can be empty. Right?
However now the `document_issuing_country` will be a mandatory field. Can you please suggest any alternate method to handle this scenario?

Reply:
And one more question related to this, can a user trade without verifying the identity?

Reply:
For CR clients, they can trade without verified

Reply:
Thanks @amina-deriv for the clarification 👍